components/appify-grid/appify-grid.component.css

```diff
@@ -1,68 +1,92 @@
 #container {
-    width: 100%;
-    color: var(--color-foreground);
-    background: var(--color-background);
-    background-position: center;
-    background-repeat: no-repeat;
-    background-size: cover;
+  width: 100%;
+  color: var(--color-foreground);
+  background: var(--color-background);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
 }
 
 #inner-container {
-    max-width: 100%;
-    margin: auto;
-    padding: calc(var(--size-section-padding) / 2) var(--spacing-xxLarge);
+  max-width: 100%;
+  margin: auto;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: flex-start;
+  /*padding: calc(var(--size-section-padding) / 2) var(--spacing-xxLarge);*/
```

Comment:
Two things in one comment:

1. Can we do 4 space tabs in this project as well all over?
2. Can we remove the commented line?